docs/docs/guides/manage/customize/branding.md

```diff
@@ -46,7 +46,7 @@ If you like to trigger your settings for your applications you have different po
 Send a [reserved scope](/apis/openidoauth/scopes) with your [authorization request](../../integrate/login/oidc/login-users#auth-request) to trigger your organization.
 The primary domain scope will restrict the login to your organization, so only users of your own organization will be able to login.
 
-You can use our [OpenID Authentication Request Playground](/apis/openidoauth/authrequest) to learn more about how to trigger an [organization's policies and branding](/apis/openidoauth/authrequest#organization-policies-and-branding).
+You can use our [OpenID Authentication Request Playground](/oidc-playground) to learn more about how to trigger an [organization's policies and branding](/oidc-playground#organization-policies-and-branding).
```

Comment:
```suggestion
You can use our [OpenID Authentication Request Playground](/playgrounds/oidc) to learn more about how to trigger an [organization's policies and branding](/oidc-playground#organization-policies-and-branding).
```

Reply:
I guess the second part of the link should be also different -> [organization's policies and branding](/oidc-playground#organization-policies-and-branding).